.teamcity/settings.kts

```diff
@@ -0,0 +1,49 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.project
+import jetbrains.buildServer.configs.kotlin.v2019_2.version
+import jetbrains.buildServer.configs.kotlin.v2019_2.DslContext
+import jetbrains.buildServer.configs.kotlin.v2019_2.AbsoluteId
+import common.Branch
+import projects.GradleBuildToolRootProject
+
+version = "2020.2"
+
+/*
+
+Master (buildTypeId: Gradle_Master)
+ |----- Check (buildTypeId: Gradle_Master_Check)
+ |        |---- QuickFeedbackLinux (buildTypeId: Gradle_Master_Check_QuickFeedbackLinux)
+ |        |---- QuickFeedback
+ |        |---- ...
+ |        |---- ReadyForRelease
+ |
+ |----- Promotion (buildTypeId: Gradle_Master_Promotion)
+ |        |----- Nightly Snapshot
+ |        |----- ...
+ |
+ |----- Util
+         |----- WarmupEc2Agent
+         |----- AdHocPerformanceTest
+
+Release (buildTypeId: Gradle_Release)
+ |----- Check (buildTypeId: Gradle_Release_Check)
+ |        |---- QuickFeedbackLinux (buildTypeId: Gradle_Release_Check_QuickFeedbackLinux)
+ |        |---- QuickFeedback
+ |        |---- ...
+ |        |---- ReadyForRelease
+ |
+ |----- Promotion (buildTypeId: Gradle_Release_Promotion)
+ |        |----- Nightly Snapshot
+ |        |----- ...
+ |
+ |----- Util
+         |----- WarmupEc2Agent
+         |----- AdHocPerformanceTest
+ */
+val branch = Branch.current()
+val parentProjectId = "Gradle"
+
+DslContext.parentProjectId = AbsoluteId(parentProjectId)
```

Comment:
I don't think we should or need to set this. This is already set correctly, right? Same goes for `projectId` and `projectName`.

We set this up manually when adding the versioned settings to a new project in Teamcity, right? There shouldn't be a need to set this manually here again.